tests/framework/web/session/AbstractDbSessionTest.php

```diff
@@ -147,6 +148,35 @@ public function testWriteCustomField()
         $this->assertSame('changed by callback data', $session->readSession('test'));
     }
 
+    /**
+     * @depends testReadWrite
+     */
+    public function testWriteCustomFieldWithUserId()
+    {
+        $session = new DbSession();
+        $session->open();
+
+        $savedUserId = mt_rand(111, 999);
```

Comment:
Is it necessary for it to be random?

Reply:
No, it just did development for me slightly easier to check that test locally passed during debug. Shall I make it static, e.g. 123 ?

Reply:
Keeping things simple is better. Yes.

Reply:
OK, done.